src/main/java/org/folio/innreach/domain/entity/InnReachLocation.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.innreach.domain.entity;
+
+import java.util.UUID;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import org.folio.innreach.domain.entity.base.Auditable;
+
+@Getter
+@Setter
+@EqualsAndHashCode(of = "code", callSuper = false)
+@ToString(callSuper = true)
+@Entity
+@Table(name = "inn_reach_location")
+public class InnReachLocation extends Auditable {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private UUID id;
+  private String code;
```

Comment:
From the requirements: 
> INN-Reach location code: limit five (5) lower-case alphanumeric characters
> INN-Reach location description: limited to 255 characters

How will the above limitations be supported?

Reply:
We have the database constraint for the location code, also we'll add REST API validation for these values in the future

Reply:
ok, agree